crates/bevy_core_pipeline/src/core_2d/mod.rs

```diff
@@ -106,7 +106,7 @@ impl PhaseItem for Transparent2d {
 
     #[inline]
     fn sort(items: &mut [Self]) {
-        items.sort_by_key(|item| item.sort_key());
+        radsort::sort_by_key(items, |item| item.sort_key().0);
```

Comment:
I'm not sure what would be useful and correct information. Maybe:
```suggestion
        // radsort is a stable radix sort that performed better than .sort_by_key() or .sort_unstable_by_key()
        radsort::sort_by_key(items, |item| item.sort_key().0);
```
@alice-i-cecile Do you think that adds anything? I'm not sure I do. That said, that TransparentUi uses a stable sort looks surprising and a comment there saying why would be useful, else it can probably also use radsort.

Reply:
Also, ECS iteration order is not guaranteed so perhaps we should use the `Entity.index()` in the sort key for consistency, e.g. `(FloatOrd, u32)`.

Reply:
I am not sure either. It makes some sense in light of the existing docs on the `PhaseItem` trait.

If every impl is not using the default sort and is instead using radsort though, maybe those docs need a pass.

Reply:
I think the main bit that I would want to document here is "why are we ignoring the outer type of the SortKey when sorting?"

But the answer: "it made for a tiny diff and most of the other impls do it" isn't very satisfying.

Reply:
I’d probably accept a PR that called .sort_key() in .sort() if that’s what you mean.

Reply:
But this feels like minor details to me.